### PR TITLE
Consistency in implicit attach and wait for sync on presence#get

### PR DESCRIPTION
**Pull request by mattheworiordan**
This addresses the proposed clarification of the implicit behaviour in https://github.com/ably/docs/issues/67

@paddybyers @SimonWoolf please review

### CODE REVIEW

File: content/client-lib-development-guide/features.textile [hunk 1]
```diff
@@ -456,9 +456,9 @@ h3(#realtime-presence). Presence
 ** @(RTP10e)@ In all other ways, this method is identical to @Presence#enter@ and should have matching tests
 * @(RTP11)@ @Presence#get@ function:
 ** @(RTP11a)@ Returns the list of current members on the channel in a callback and, by default, will not wait for the @SYNC@ to be completed
```
Comment (SimonWoolf) [inline content/client-lib-development-guide/features.textile]: `will not wait` should now be `will wait`

**Comment by SimonWoolf** [comment]
"FAILED state" wording should be added to RTP6c (presence#subscribe) too.

:+1:, looks good!

File: content/client-lib-development-guide/features.textile (hunk 1 above)
Comment (mattheworiordan) [inline content/client-lib-development-guide/features.textile]: Thanks, will fix

**Comment by mattheworiordan** [comment]
@SimonWoolf have addressed your issues, thanks for casting your eye over it :)

**Comment by paddybyers** [comment]
LGTM.

**Pull request merged**